search/includes/classes/class-queue.php

```diff
@@ -471,10 +471,36 @@ public function process_jobs( $jobs ) {
 			// Mark them as done in queue
 			$this->delete_jobs( $jobs );
 
+			$this->record_processed_from_queue_stat( count( $jobs ), $indexable );
+
 			$this->record_queue_count_stat( $indexable );
 		}
 	}
 
+	public function record_processed_from_queue_stat( $count, $indexable ) {
+		if ( ! $indexable ) {
+			return;
+		}
+
+		if ( ! is_int( $count ) ) {
+			$count = intval( $count );
```

Comment:
Should this instead return? If a non-int came in, we probably don't want to rely on PHP typecasting for stat values.

Reply:
I don't think it makes much of a difference. Last time we had the same conversation on different sides 😄.

[intval on invalid values return 0](https://www.php.net/manual/en/function.intval.php#refsect1-function.intval-returnvalues). Incrementing by 0 doesn't do anything, so I think it's fine.

Reply:
Alright, LGTM then